quantus/metrics/robustness/relative_representation_stability.py

```diff
@@ -0,0 +1,335 @@
+# This file is part of Quantus.
+# Quantus is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.
+# Quantus is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more details.
+# You should have received a copy of the GNU Lesser General Public License along with Quantus. If not, see <https://www.gnu.org/licenses/>.
+# Quantus project URL: <https://github.com/understandable-machine-intelligence-lab/Quantus>.
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Iterable
+import numpy as np
+from functools import partial
+
+if TYPE_CHECKING:
+    from typing import Optional, Callable, Dict, List
```

Comment:
The user might not have tf or torch installed with Quantus, wouldn't it fail for them then to import it in this way?

See explanation_func for current workaround